Radzen.Blazor/RadzenCompareValidator.cs

```diff
@@ -30,7 +30,7 @@ private int Compare(object componentValue)
             switch (componentValue)
             {
                 case String stringValue:
-                    return String.Compare(stringValue, (string)Value, false, CultureInfo.CurrentCulture);
+                    return String.Compare(stringValue, (string)Value, false, GetComponentCulture());
```

Comment:
Why not just `Culture`? What is the purpose of the `GetComponentCulture` method?

Reply:
That's right, `GetComponentCulture()` was the original method in the first commit to get the specified/default/currentculture and made a virtual method to make it like the GetComponentCss()/etc. methods, but I changed it later to resolve on the property itself to allow it to work with `IScheduler` (instead of creating a specific interface implementation) and left that there. I'll remove it.

By the way, I've noticed that `IScheduler` has a setter on the `Culture` property which is not used (and I believe that should be removed from the interface). This is generally bad in Blazor because changing a parameter property directly on the property (and not through a parameter change) does not raise `SetParametersAsync` nor any of the parameter-related component lifecycle methods.

Removing the setter from the interface does not seem to bear any consequences (because it's only used for reading in child components). Should we remove that setter, while we're at it?